workitem/field_definition_blackbox_test.go

```diff
@@ -36,3 +25,53 @@ func TestListFieldDefMarshalling(t *testing.T) {
 		t.Errorf("field should be %v, but is %v", def, unmarshalled)
 	}
 }
+
+func TestFieldDefinition_Marshalling(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.UnitTest)
+
+	t.Run("simple boolean type", func(t *testing.T) {
+		t.Parallel()
+		def := FieldDefinition{
+			Required:    true,
+			Label:       "Salt",
+			Description: "Put it in your soup",
+			Type: SimpleType{
+				Kind: KindBoolean,
+			},
+		}
+		testFieldDefinitionMarshalUnmarshal(t, def)
+	})
+
+	t.Run("list type", func(t *testing.T) {
```

Comment:
@aslakknutsen I've re-written the test file and there we only tested the list field type but not a simple and not the enum list type. The enum test is unrelated, yes, but the list type previously existed.